ci/prow-entrypoint.sh

```diff
@@ -114,8 +114,8 @@ cosa_build() {
 kola_test_qemu() {
     cosa buildextend-qemu
     cosa kola --basic-qemu-scenarios
-    kola run-upgrade -b rhcos -v --find-parent-image --qemu-image-dir tmp/ --output-dir tmp/kola-upgrade
-    cosa kola run --parallel 2
+    # kola run-upgrade -b rhcos -v --find-parent-image --qemu-image-dir tmp/ --output-dir tmp/kola-upgrade
```

Comment:
Let's remove this line entirely, here and in other branches as well (start from master and go down).